.vscode/settings.json

```diff
@@ -104,7 +100,5 @@
     "NODE_ENV": "development"
   },
   "typescript.tsdk": "common/temp/node_modules/typescript/lib",
-  "cSpell.words": [
-    "queryables"
-  ],
-}
\ No newline at end of file
+  "cSpell.words": ["queryables"]
```

Comment:
move this to the cspell file, and revert changes to this file since they all come from prettier formatter

Reply:
moved, and reverted the other formatting